horse/repositories.py

```diff
@@ -0,0 +1,38 @@
+from uuid import uuid4
+
+from horse.models import User, Movie
+
+
+class ObjectDoesNotExist(Exception):
+    def __init__(self, model, pk):
+        super().__init__('{}: {}'.format(model, pk))
+
+
+class InMemoryRepository:
```

Comment:
I think it would be a good idea to have abstract repository class that defines repository interface